roofit/roofitcore/src/RooAbsArg.cxx

```diff
@@ -2374,16 +2388,21 @@ void RooAbsArg::SetNameTitle(const char *name, const char *title)
 void RooAbsArg::Streamer(TBuffer &R__b)
 {
    if (R__b.IsReading()) {
-     _ioReadStack.push(this) ;
+     ioReadStack().push(this) ;
```

Comment:
This is not thread safe.  2 threads reading a RooAbsArg will trample on each other.

Reply:
But the 'failure' mode may simply be data put fine in the wrong place leading to inaccurate results that the user may or may not notice and may or may not guess where it is form ... i.e. a silent failure :(.